packages/frontend/core/src/utils/__tests__/extract-emoji-icon.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from 'vitest';
+
+import { extractEmojiIcon } from '../extract-emoji-icon';
+
+test('extract-emoji-icon', () => {
+  expect(extractEmojiIcon('👨🏻‍❤️‍💋‍👨🏻123')).toEqual({
+    emoji: '👨🏻‍❤️‍💋‍👨🏻',
+    rest: '123',
+  });
+
+  expect(extractEmojiIcon('❤️123')).toEqual({
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/5f22e9c8-5306-4daf-a98a-27b43ffcab0e.png)

it seems ❤️ displays as emoji on windows

Reply:
Its depending on the font family, we adopt a conservative strategy, only exclusive emoji characters

Reply:
yeah i mean maybe we could use other symbols that reduces confusion here